ooniprobe/Utility/NetworkSession.h

```diff
@@ -0,0 +1,17 @@
+//
+//  NetworkSession.h
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 16/03/2020.
+//  Copyright © 2020 OONI. All rights reserved.
```

Comment:
Can you make this header look like other files or emit it? The way in which it is now makes the file not open source.